runtime/src/lib.rs

```diff
@@ -978,7 +978,7 @@ construct_runtime! {
 		Schema: pallet_schema = 103,
 		Registry: pallet_registry = 104,
 		Stream: pallet_stream = 105,
-		Scoring: pallet_scoring = 163,
+		Scoring: pallet_scoring = 106,
 		DidNames: pallet_did_names = 106,
```

Comment:
Should both `Scoring` and `DidNames` have the same number `106`? What is this number btw ?

Reply:
It can't have the same number. This number represents the pallet index. A new pallet should take the next available number in the series. in this case scoring pallet number should be 107.

Reply:
Seems I have caught an early bug.